assets/js/modules/adsense/datastore/sites.test.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * `modules/adsense` data store: sites tests.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import { MODULES_ADSENSE } from './constants';
+import { CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import {
+	createTestRegistry,
+	subscribeUntil,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+import * as fixtures from './__fixtures__';
+
+describe( 'modules/adsense sites', () => {
+	let registry;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	describe( 'actions', () => {} );
+
+	describe( 'selectors', () => {
+		describe( 'getSites', () => {
+			it( 'uses a resolver to make a network request', async () => {
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/modules\/adsense\/data\/sites/,
+					{ body: fixtures.sites, status: 200 }
```

Comment:
We don't need `status: 200` since the status is 200 by default.
```suggestion
					{ body: fixtures.sites }
```

Reply:
I'll merge this as-is, but in the future I don't think removing this explicitness, especially in a test, is necessary. It gives extra immediate context to the person reading the test and is supported by the API, so I think it's nice to leave it 🙂